Recurly/Resources/AccountAcquisitionCost.cs

```diff
@@ -17,10 +20,12 @@ public class AccountAcquisitionCost : Request
 
         /// <value>The amount of the corresponding currency used to acquire the account.</value>
         [JsonProperty("amount")]
+
```

Comment:
I think these probabaly should sit right on top of their attribute they are referring to

Reply:
I fixed this recently